public/stylesheets/styles.css

```diff
@@ -1,7 +1,78 @@
-/*
-white: #f7f9f9;
+
+/* white: #f7f9f9;
 light-blue: #2081c3;
 dark-blue: #0a2c42;
 grey: #bfbfbf;
-font-family: Arial, Helvetica, sans-serif;
-*/
+font-family: Arial, Helvetica, sans-serif; */
+
+.container-div{
+    width: 100wv;
+    height: 91vh;
+}
+
+.buttons {
+    display: block;
+    text-align: center;
+    width: 215px;
+    height: 25px;
+    margin: 20px auto 0px auto;
+    background-color:#2081c3;
+    color: white;
+    border-radius: 2px;
+    text-decoration: none;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+img {
+    display: flex;
+    margin: 0 auto;
+}
+
+.footer {
+
```

Comment:
Remove extra spaces here. Same thing on next lines.

Not a big deal now but starting to take the habit of writing clean and organised code will definitely help you in the long run.

Reply:
Sure, that's something I have to work on..